### ISSUE TITLE
Network delay causes input text to get reset

### ISSUE THREAD

**Issue by eruvanos** [mikeabrahamsen/Flask-Meld#14]
Running the example from https://github.com/mikeabrahamsen/Flask-Meld-Example works well on localhost.

Deploying it to some cloud infrastructure results in some uncanny behaviour.
Due to the network lag, text typed (fast) into an input field gets overwritten by old state when the new html is rendered into the field. 

This can be emulated locally by adding `sleep(0.5)` into the render methods like

```
def render(self, component_name):
    sleep(0.5)
    return super().render(component_name)
```

Right now I have no idea for a workaround :/

**Comment by eruvanos** [comment]
I read a little bit more and found everything in the [livewire docs](https://laravel-livewire.com/docs/2.x/properties#debouncing-input). I see that most of the features are already prepared in the element class.

**Comment by mikeabrahamsen** [comment]
This is on my radar, I have been working through a few different ways of solving this. The debounce is certainly one way of doing it. I'll do a bit of prototyping this weekend!

**Comment by mikeabrahamsen** [comment]
With your help the debounce functionality has been implemented 4fd99bcf88bd38a8dd227107d8c75763b8661a99
This issue will be closed. I would love to talk to you more about how you deployed the application. Please join us on the Flask-Meld discord: https://discord.gg/mfKEqKrCB9
